morphir/runtime/src/org/finos/morphir/runtime/quick/testing/UnitTesting.scala

```diff
@@ -47,7 +51,7 @@ object UnitTesting {
       val testNames = collectTests(globals)
       val testIRs   = testNames.map(fqn => Value.Reference.Typed(testType, fqn))
       if (testIRs.isEmpty) {
-        val emptySummary = TestSummary("No tests run", Map())
+        val emptySummary = TestSummary(CoverageInfo(List.empty[FQName], List.empty[FQName]), "No tests run", Map())
```

Comment:
Only one of these should be empty - `userDefinedFunctions` should be defined above this, and the empty summary should still express the full list of functions. (I also think this implementation will cause div/null errors as is, but I'm not certain)